pkgs/tools/misc/ethminer/default.nix

```diff
@@ -8,29 +9,30 @@
   boost,
   makeWrapper,
   cudatoolkit,
+  withCuda ? cudatoolkit != null,
```

Comment:
```suggestion
  withCuda ? true,
```

Reply:
I'm not sure what the benefit of that would be?

My way makes it a bit more explicit why cudatoolkit (unfree) is needed. When calling the package from a Nixpkgs which doesn't have unfree packages in it for example, it'd default to a free build which is a useful property I think.

Reply:
This is really badly documented, but the default name for this flag now seems to be `cudaSupport, with its value set to `config.cudaSupport`:

https://github.com/NixOS/nixpkgs/blob/350f9bd822e1c5017482e6364dbafda36ff4157a/pkgs/top-level/all-packages.nix#L1723

This has the benefit that if someone already has `cudaSupport = true; allowUnfree = true` in their nixpkgs `config`, CUDA support would also be enabled automatically for ethminer.